lib/collection/src/operations/cluster_ops.rs

```diff
@@ -28,22 +28,16 @@ pub enum ClusterOperations {
     RestartTransfer(RestartTransferOperation),
 
     /// Start resharding
-    #[schemars(skip)]
     StartResharding(StartReshardingOperation),
     /// Finish migrating points on specified shard, mark shard as `Active`
-    #[schemars(skip)]
     FinishMigratingPoints(FinishMigratingPointsOperation),
     /// Commit read hashring
-    #[schemars(skip)]
     CommitReadHashRing(CommitReadHashRingOperation),
     /// Commit write hashring
-    #[schemars(skip)]
     CommitWriteHashRing(CommitWriteHashRingOperation),
     /// Finish resharding
-    #[schemars(skip)]
     FinishResharding(FinishReshardingOperation),
```

Comment:
Do we actually want to expose these in the OpenAPI? Users only need to know about start and abort.

Reply:
GitHub crapped itself, this should have selected
- `FinishMigratingPoints`
- `CommitReadHashRing`
- `CommitWriteHashRing`
- and `FinishResharding`

Reply:
Yeah, it may be better to hide these except start/abort resharding.